src/app/services/security/http-interceptor.service.ts

```diff
@@ -21,9 +21,24 @@ export class HttpinterceptorService implements HttpInterceptor {
     const tempToken = auth.getToken();
 
     if (tempToken != null) {
-      const afterTokenreq: HttpRequest<any> = req.clone({
-        setHeaders: { authorization: tempToken },
+      const afterTokenreq: HttpRequest<any>;
+
+        // CSRF
+        if (environment.security === 'csrf') {
+          authReq = req.clone({
```

Comment:
You have used `authReq` in place of `afterTokenreq` (Check line number 24)

Reply:
Ok